### DISCUSSION TITLE
Homematic (IP) local

### DISCUSSION THREAD

**Discussion by Fulleranio** [OpenCCU/OpenCCU#2576] [🇩🇪 Fragen&Antworten]
Here I am, having installed Homematic (IP) local successfully and being able to handle all the devices installed. I have only one problem left: the devices appear double in the overview, one from the rapsberrymatic and one from the newly installed Homematic (IP) local. I guess, all I hav to do is to delete the entities which derive from the raspberrymatic, but I dont know how since the entities are all connected.

**Comment by MichaelN0815** [comment]
Kannst du das etwas klarer beschreiben und vielleicht mal einen Screenshot zeigen?

**Reply by Fulleranio** [reply]
Hallo Michael,
ja, 
![HomeAssRoll MW-WZ-o](https://github.com/jens-maus/RaspberryMatic/assets/136200548/2f977e47-e5bc-45c7-861a-822e64385188)
![RaspberryMatic – Rolladen MW-o](https://github.com/jens-maus/RaspberryMatic/assets/136200548/38307843-01f9-447e-b6dd-e94d34519e47)


ich hoffe, die Dateien sind zu sehen. Die Installation wurde zunächst als Raspberrymatic und nachdem das funktionierte als Add-on in Home Assistant installiert. Die Übernahme der Sicherung in das Add-on funktionierte einwandfrei, die Geräte waren sofort in Home Assistant bedienbar übernommen. Ausnahme war das DRAP, das aber mit dem Patch nun auch einwandfrei funktioniert. Leider tauchten nach Installation der Integration "Homematic (IP) local" nun Entitäten in der Übersicht auf, die schon vor der Installation von "Homematic (IP) local" vorhanden waren und dadurch nun doppelt in der Übersicht vorhanden sind. Im Bild (HomeAssRoll MW-WZ-o) sind die Rolladen "WZ Mitte Roll ost" und "Ost" ein und dieselben Aktoren, "Ost" reagiert nicht, "WZ Mitte Roll ost" funktioniert einwandfrei. Die dazugehörigen Entitäten heißen in Home Assistant "cover.wz_mitte_roll_ost" und "cover.qeq1923576". Wenn ich nun herausfinde, wie ich die überflüssigen Entitäten entfernen kann, ist alles in Ordnung, denke ich.
Danke im Vorraus!

**Reply by MichaelN0815** [reply]
Ah, ich habe nur Ahnung von RaspberryMatic. Da kann ich nicht weiter helfen.
